packages/flutter/test/material/chip_test.dart

```diff
@@ -188,6 +188,91 @@ void _expectCheckmarkColor(Finder finder, Color color) {
   );
 }
 
+Future<void> pumpFrames({
+  WidgetTester tester,
+  int frames,
+  Duration duration,
+}) async {
+  int i;
+  for (i = 0; i < frames; i ++) {
+    await tester.pump(duration);
```

Comment:
i++

Reply:
Fixed to `++i`; see discussion with gspencergoog below.